lib/ui/platform_dispatcher.dart

```diff
@@ -2605,3 +2662,52 @@ class SemanticsActionEvent {
     );
   }
 }
+
+/// Signature for [PlatformDispatcher.onPlatformFocusChange].
+typedef ViewFocusChangeCallback = void Function(ViewFocusEvent);
+
+/// An event for the engine to communicate view focus changes back to the app.
+final class ViewFocusEvent {
+  /// Creates a [ViewFocusChange].
+  const ViewFocusEvent({
+    required this.viewId,
+    required this.focusState,
+  });
+
+  /// The id of the [FlutterView] that experienced a focus change.
+  final int viewId;
+
+  /// The current [ViewFocusState] of the [FlutterView] with the given [viewId].
+  final ViewFocusState viewFocusState;
+}
+
+/// Represents a given flutter view focus state.
```

Comment:
leave a breadcrumb back to ViewFocusEvent and potentially the onPlatformFocusChange API in this doc.

Reply:
Done